prov/sm2/src/sm2.h

```diff
@@ -356,4 +356,14 @@ static inline size_t sm2_pop_xfer_entry(struct sm2_ep *ep,
 	return FI_SUCCESS;
 }
 
+static void inline sm2_freestack_push(struct sm2_ep *ep,
+				      struct sm2_xfer_entry *xfer_entry)
+{
+	struct sm2_av *av =
+		container_of(ep->util_ep.av, struct sm2_av, util_av);
+	struct sm2_mmap *map = &av->mmap;
+	struct sm2_region *region = sm2_mmap_ep_region(map, ep->gid);
+	smr_freestack_push(sm2_freestack(region), xfer_entry);
```

Comment:
Is `smr_freestack_push` in `ofi*.h`? The naming looks for shm only

Reply:
yes, it is in ofi_mem.h.  The naming is... unfortunate to me.

Reply:
I have a backlog item in my notes to get us off of the SMR freestack... but yes, it should be renamed too.